.github/workflows/make_bundle.yml

```diff
@@ -5,6 +5,9 @@ on:
       - "v*" # Push events to matching v*, i.e. v1.0, v20.15.10
     branches:
       - master
+  pull_request:
+    branches:
+      - master
```

Comment:
Did this change mean we're now making the bundle on every PR? Is this something we want to keep doing @tlambert03 @jni?

Reply:
I only added this temporarily to have some feedback in the PR itself. I don't think it's needed for every PR and can be reverted safely.

Reply:
Hm, maybe it's ok to leave it here? It caught a bug on Windows bundles in [an unrelated PR](https://github.com/napari/napari/pull/3297#issuecomment-909063538).